### PR TITLE
Use the correct tray menu ID for the container runtime.

### LINKED ISSUE #3037: Wording inconsistent between Container Engine and Container Runtime
### Actual Behavior

Tray menu: 
image

Preferences window:
image

### Steps to Reproduce

1. Open preferences window.
2. Open tray menu.

### Result

The name for "the thing that runs containers" is inconsistent between the two places.

### Expected Behavior

Pick one phrase and stick to it throughout the application.

### Additional Information

_No response_

### Rancher Desktop Version

1.5.0-701-g2d00a573

### Rancher Desktop K8s Version

N/A

### Which container engine are you using?

containerd (nerdctl)

### What operating system are you using?

Windows

### Operating System / Build Version

Windows 11 Home 21H2 (Build 22000.1042)

### What CPU architecture are you using?

x64

### Linux only: what package format did you use to install Rancher Desktop?

_No response_

### Windows User Only

N/A

### LINKED ISSUE #3039: 1.6.0 RC1-Tray menu not showing correct container runtime
### Actual Behavior

After switching container engine from containerd to docker, the tray menu still shows containerd as container runtime.

### Steps to Reproduce

Test case 20

1. Start RD - select containerd runtime
2. Check Tray menu for container runtime: Containerd
3. Switch to Docker container runtime from preferences
4. Check Tray menu for container runtime

### Result

Tray menu shows containerd container runtime 

### Expected Behavior

Tray menu should update with change in container runtime

### Additional Information

Screen Shot 2022-09-27 at 4 28 48 PM

### Rancher Desktop Version

1.5.0-701-g2d00a573

### Rancher Desktop K8s Version

1.18.20

### Which container engine are you using?

moby (docker cli)

### What operating system are you using?

macOS

### Operating System / Build Version

macOS Catalina

### What CPU architecture are you using?

x64

### Linux only: what package format did you use to install Rancher Desktop?

_No response_

### Windows User Only

_No response_

### PR DESCRIPTION
**Pull request by ericpromislow**
Fixes #3037
Fixes #3039

Signed-off-by: Eric Promislow <epromislow@suse.com>

### CODE REVIEW

**Comment by ericpromislow** [comment]
Note that there are instances of `/container.*runtime/i` in the code, but there are three sources:
* bats
* localizations most likely inherited from dashboard
* The `--container-runtine-endpoint` flag for k3s

None of these affect this PR's issues.

**Review by rak-phillip — APPROVED** [review]
LGTM - waiting on CI to finish before merging

**Pull request merged**
